sdk/security/keyvault/azadmin/rbac/client_test.go

```diff
@@ -48,7 +45,13 @@ func TestRoleDefinition(t *testing.T) {
 	// test create definition
 	createdDefinition, err := client.CreateOrUpdateRoleDefinition(context.Background(), scope, name, parameters, nil)
 	require.NoError(t, err)
-	require.Equal(t, name, *createdDefinition.Name)
+
+	if recording.GetRecordMode() == recording.PlaybackMode {
```

Comment:
I suggest either removing the `name` sanitizer (should be [this one](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs#L651), names in these tests aren't secret) or asserting only in recording & live modes

Reply:
fixed in [bdf929b](https://github.com/Azure/azure-sdk-for-go/pull/22911/commits/bdf929b94d18bf715cd10920a3560ad04f310d87)[bdf929b](https://github.com/Azure/azure-sdk-for-go/pull/22911/commits/bdf929b94d18bf715cd10920a3560ad04f310d87)